udacity/log-analysis-project/Readme.md

```diff
@@ -1,16 +1,31 @@
-You will need to unzip **newsdata.sql.zip** file after downloading it. The file inside is called **newsdata.sql** Put this file into the [vagrant](https://www.vagrantup.com/downloads.html) directory, which is shared with your virtual machine.
+In this project we work with the database.The database contains newspaper articles, as well as the web server log for the site. The log has a database row for each time a reader loaded a web page. Using that information, code answer questions about the site's user activity.
+
+You need to use a virtual machine (VM) to run an SQL database server.
+
+VirtualBox is the software that actually runs the virtual machine. You can download it from [HERE](virtualbox.org). Install the platform package for your operating system.
+
+Vagrant is the software that configures the VM and lets you share files between your host computer and the VM's filesystem. Download it from [vagrantup.com](https://www.vagrantup.com/downloads.html).
```

Comment:
Virtualbox itself handles the sharing, not Vagrant.

Reply:
Noted!